wgpu-types/src/lib.rs

```diff
@@ -7243,3 +7245,154 @@ pub enum DeviceLostReason {
     /// will call the callback immediately, with this reason.
     DeviceInvalid = 4,
 }
+
+/// An internal counter for debugging purposes
```

Comment:
I know wgpu-types at this point has a long tradition of only being a single monstrous lib.rs file, but this seems like a nice reason to add  `counters.rs` to make things more readable

Reply:
+1 to not having a monstrous `lib.rs` file. This has made some things simple, but also, editor operations on this file feel heavy.